client/client.go

```diff
@@ -195,12 +195,18 @@ func (client *Client) ServerVersion() (string, error) {
 		return "unknown", err
 	}
 
-	version := sysInfo.Version
-	if version == "" {
-		version = "unknown"
+	serverVersion := sysInfo.Version
+	if serverVersion == "" {
+		serverVersion = "unknown"
 	}
 
-	return fmt.Sprintf("%s (series %s)", version, sysInfo.Series), nil
+	versionStr := fmt.Sprintf(""+
+		"snapd:  %s\n"+
+		"series: %s\n", serverVersion, sysInfo.Series)
```

Comment:
looking at the suggestion it seems the colons shouldn't be there?  also not sure if series was meant to be on its own line or same line as snapd

also given the new lines wondering if this should now return []string or even [][2]string instead and let the caller do the separations as it wants